app/models/mixins/relationship_mixin.rb

```diff
@@ -653,6 +653,14 @@ def replace_children(*child_objs)
     child_objs = child_objs.flatten
     return remove_all_children if child_objs.empty?
 
+    if use_ancestry?
+      to_add = child_objs.collect(&:id) - children.collect(&:id)
+      to_add, to_keep = child_objs.partition { |c| to_add.include?(c.id) }
```

Comment:
Can this be
```
to_add = child_objs.collect(&:id) - children.collect(&:id)
to_remove = children.collect(&:id) - child_objs.collect(&:id)
```

Reply:
it can't take ids, it needs objects

Reply:
Ah didn't realize the first to_add was ids and the second was objects, the rename makes it more clear thanks